targets/kde

```diff
@@ -9,7 +9,12 @@ CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal kde-plasma-desktop -- network-manager
+if release -lt vivid then
```

Comment:
Whoops, nice catch.
I need better glasses, and @imakiro needs to test his patches :)